README.md

```diff
@@ -17,30 +17,31 @@
 |                          [🐞 Report a bug or 🙋‍♂️ request a feature][issues-url]                           |
 | [![contributions welcome][contributions-welcome]][issues-url] [![License][badge-apache]][apache-license] |
 
-- [Azordev Art Marketplace](#azordev-art-marketplace)
-  - [The Project](#the-project)
+- [Art Marketplace](#art-marketplace)
+  - [Art Marketplace Docs](#art-marketplace-docs)
     - [Features](#features)
     - [Built With](#built-with)
+  - [Pre requirements](#pre-requirements)
+  - [How to run the project:](#how-to-run-the-project)
   - [Create a .env file](#create-a-env-file)
-  - [Advantages Of Styled-Components](#advantages-of-styled-components)
-  - [Using the App](#using-the-app)
-  - [How to run this project](#how-to-run-this-project)
-    - [Prerequisites](#prerequisites)
-    - [Run it (Quickstart with docker)](#run-it-quickstart-with-docker)
-    - [Installing the project](#installing-the-project)
+  - [Technologies:](#technologies)
+  - [Advantages Of TailwindCSS](#advantages-of-tailwindcss)
+  - [File Structure:](#file-structure)
+  - [GIT USE:](#git-use)
+  - [FAQ:](#faq)
     - [Available Scripts](#available-scripts)
       - [`npm start`](#npm-start)
       - [`npm run build`](#npm-run-build)
     - [Learn More](#learn-more)
   - [Potential Features](#potential-features)
-  - [Author](#author)
+  - [Collaborators](#collaborators)
   - [Contributing](#contributing)
   - [Show your support](#show-your-support)
   - [License](#license)
 
-## The Project
+## Art Marketplace Docs
 
-A place where you can find a download BIM objects
+A place where you can find a download Art images
```

Comment:
Maybe mention here that we use the endpoint for real art

Reply:
Can ai ask for some kind of icon or image @angelik0828? To use as favicon and to put in this document

Reply:
@Katsu08 we can add this image later in other issue